developers/weaviate/configuration/compression/bq-compression.md

```diff
@@ -167,6 +167,12 @@ For example:
 
 ## Multiple vector embeddings (named vectors)
 
+import NamedVectorCompress from '/_includes/named-vector-compress.mdx';
+
+<NamedVectorCompress />
+
+## Multi-vector embeddings (ColBERT, ColPali, etc.)
```

Comment:
Somewhat on the side: do you think people might find "multiple vector embeddings" vs "multi-vector embeddings" confusing? I wonder if we should consider renaming "multiple vector embeddings". 

Maybe to "multiple object vectors" or "multiple vector representations"

Reply:
I think we can leave it as is for now to be consistent with the rest of the docs because the parentheses (named vectors) and (ColBERT, ColPali, etc.) hopefully provide enough context. However, we should revisit this at a later point if someone points it out as confusing.